java/steps/yaks-standard/src/main/java/org/citrusframework/yaks/hooks/InjectEnvVarsHook.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.citrusframework.yaks.hooks;
+
+import com.consol.citrus.TestCaseRunner;
+import com.consol.citrus.actions.AbstractTestAction;
+import com.consol.citrus.annotations.CitrusResource;
+import com.consol.citrus.context.TestContext;
+import io.cucumber.java.Before;
+import io.cucumber.java.Scenario;
+
+/**
+ * Cucumber hook injects environment variables as test variables before the scenario is executed.
+ * @author Christoph Deppisch
+ */
+public class InjectEnvVarsHook {
+
+    @CitrusResource
+    private TestCaseRunner runner;
+
+    @Before
+    public void injectEnvVars(Scenario scenario) {
+        runner.run(new AbstractTestAction() {
+            @Override
+            public void doExecute(TestContext context) {
+                context.setVariable("SCENARIO_NAME", scenario.getName());
+                context.setVariable("SCENARIO_ID", scenario.getId());
+
+                injectEnvVar(context, "CLUSTER_WILDCARD_DOMAIN", "ns.svc.cluster.local");
```

Comment:
`ns.svc.cluster.local` was just an example and the `ns` is a shortcut for `namespace`. It should be `${YAKS_NAMESPACE}.svc.cluster.local`

Reply:
haha I thought that would be a no-brainer so I started to switch of mine. Thanks for pointing to it - I will fix that

Reply:
@bouskaJ solved